.devcontainer/addcodespacename.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
```

Comment:
Personal preference, I would add this to the top of the script, so that if any other the command in this script fail to run, the entire script fails (making it easier to debug later on!).


```suggestion
set -e
```

For more info on these flags see https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script